.ci/jobs/elastic+terraform-provider-ec+pull-request.yml

```diff
@@ -0,0 +1,22 @@
+---
+- job:
+    name: elastic+terraform-provider-ec+pull-request
+    display-name: elastic / terraform-provider-ec - pull-request
+    description: Pull request testing
+    project-type: pipeline
+    concurrent: true
+    triggers:
+        - github-pull-request:
+            github-hooks: true
+            org-list:
+                - elastic
+            allow-whitelist-orgs-as-admins: true
+            cancel-builds-on-update: true
+            status-context: acceptance
+    pipeline-scm:
+        scm:
+            - git:
+                branches:
+                    - $ghprbActualCommit
+                refspec: +refs/pull/*:refs/remotes/origin/pr/*
+                wipe-workspace: "False"
```

Comment:
Does this work? I would have thought `false` was right but maybe jjb(b) is doing some type magic 🤷‍♂️

Reply:
I'm not too sure about this one, it was a c/p from other projects settings. I don't think it even matters here for what we're doing, I'll remove it.

Reply:
It defaults to `true` and just means "don't bother cleaning the workspace since we use ephemeral workers", but there's also no harm in cleaning it, so it doesn't really matter either way I guess!